### ISSUE TITLE
about accuracy

### ISSUE THREAD

**Issue by yyl199655** [yxu0611/Tensorflow-implementation-of-LCNN#5]
I wan't to train a Feature Extractor use your model，When should I stop training?
I use your model the accuracy in training set is  about 80% no longer improve ，so what should I do?
